src/modules/account/components/BackupRecoveryPhraseFlow/BackupRecoveryPhraseFlow.js

```diff
@@ -36,10 +36,10 @@ const BackupRecoveryPhraseFlow = ({ history }) => {
           <EnterPasswordForm
             onEnterPasswordSuccess={onEnterPasswordSuccess}
           />
-          <SavePassphrase passphrase={passphrase} />
+          <SavePassphrase passphrase={passphrase} title='Back up your secret recovery phrase' />
```

Comment:
please translate it here
```suggestion
          <SavePassphrase passphrase={passphrase} title={t('Back up your secret recovery phrase')} />
```